docs/resources/permissions.md

```diff
@@ -92,7 +92,7 @@ resource "databricks_permissions" "policy_usage" {
 
 ## Instance Pool usage
 
-[Instance Pools](instance_pool.md) access control [allows to](https://docs.databricks.com/security/access-control/pool-acl.html) assign `CAN_ATTACH_TO` and `CAN_MANAGE` permissions to users and groups. It's also possible to grant creation of Instance Pools to individual [groups](group.md#allow_instance_pool_create) and [users](user.md#allow_instance_pool_create).
+[Instance Pools](instance_pool.md) access control [allows to](https://docs.databricks.com/security/access-control/pool-acl.html) assign `CAN_ATTACH_TO` and `CAN_MANAGE` permissions to users, service principals and groups. It's also possible to grant creation of Instance Pools to individual [groups](group.md#allow_instance_pool_create) and [users](user.md#allow_instance_pool_create), [service principals](service_principal.md#allow_instance_pool_create).
```

Comment:
to use consistent punctuation, please add coma before ` and`

Reply:
Hi @nfx , I'm a bit lost on these comments because I'm not english native speaker, can you maybe directly push to my branch, if you don't have access to push please let me know.